paf-mvp-core-js/src/crypto/signer.ts

```diff
@@ -0,0 +1,20 @@
+import { PrivateKey } from '@core/crypto/keys';
+import { SigningDefinition } from '@core/crypto/signing-definition';
+
+/**
+ * Class to sign data or a message
+ * S = Signed type (used to verify signature)
```

Comment:
T instead of S

Reply:
Good catch, thanks!
Will update in another PR.